opencontext_py/apps/ocitems/identifiers/ezid/ezid.py

```diff
@@ -110,7 +110,7 @@ def parse_anvl_str(self, anvl):
         metadata = LastUpdatedOrderedDict()
         for anvl_line in anvl.decode("UTF-8").splitlines():
             if ':' in anvl_line:
-                line_ex = anvl_line.split(':')
+                line_ex = anvl_line.split(':', maxsplit=1)
```

Comment:
Oh! That looks like a good fix for parsing key/values, where the value may also contain a ":" character.